fre/yamltools/__init__.py

```diff
@@ -1,9 +0,0 @@
-''' for fre.yamltools imports '''
```

Comment:
see tests, they all work!

the one interesting subcase that needed a non-empty `__init__.py` was `fre/cmor/__init__.py`. In this case, it lets one import `fre.cmor.cmor_run_subtool` instead of `fre.cmor.cmor_mixer.cmor_run_subtool`. i.e. convenience/brevity/namespace-stuff